docs/creating-a-theme.rst

```diff
@@ -90,7 +90,7 @@ First, we create a site with some content in it. We’ll use the ``nikola init``
 
     Please choose the correct time zone for your blog. Nikola uses the tz database.
     You can find your time zone here:
-    http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```

Comment:
This should also be changed in `nikola/plugins/command/init.py`.

Reply:
Unless I'm missing something, the relevant link in ``nikola/plugins/command/init.py`` already uses HTTPS.

Reply:
My bad, it is correct in `init.py`.